products/squid.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Squid
+category: server-app
+permalink: /squid
+alternate_urls:
+-   /squid-cache
+versionCommand: squid -v
+releasePolicyLink: https://wiki.squid-cache.org/ReleaseSchedule
+changelogTemplate:
+  http://www.squid-cache.org/Versions/v{{'__RELEASE_CYCLE__'|split:'.'|first}}/__RELEASE_CYCLE__/
+
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   repology: squid
+-   purl: pkg:deb/debian/squid
+-   purl: pkg:deb/ubuntu/squid
+-   purl: pkg:rpm/amzn/squid
+-   purl: pkg:rpm/redhat/squid
+-   purl: pkg:rpm/centos/squid
+-   purl: pkg:apk/alpine/squid
+
+# https://rubular.com/r/mwE2FvyQrDjXzx
+# v4+ has stable releases as major.minor
+# v2,3 had stable releases as major.minor.patch, where patch=0 was for RC releases.
+# v2 sources are now archived in a separate repo, we use that as well
+auto:
+-   git: https://github.com/squid-cache/squid.git
+    regex:
+      ^SQUID_((?<major>(2|3))_(?<minor>\d)_((STABLE)?(?<patch>\d+))|(?<major>[4-9])_(?<minor>\d+))$
+-   git: https://github.com/squid-cache/squid2.git
+    regex:
+      ^SQUID_((?<major>(2|3))_(?<minor>\d)_((STABLE)?(?<patch>\d+))|(?<major>[4-9])_(?<minor>\d+))$
+
+releases:
+-   releaseCycle: "5"
+    releaseDate: 2021-07-31
+    link: http://www.squid-cache.org/Versions/v5/squid-__LATEST__-RELEASENOTES.html
+    eol: false
+    latest: '5.9'
+    latestReleaseDate: 2023-05-01
+
+-   releaseCycle: "4"
+    releaseDate: 2018-07-03
+    eol: 2021-10-04
+    link: http://www.squid-cache.org/Versions/v4/squid-__LATEST__-RELEASENOTES.html
+    latest: '4.17'
+    latestReleaseDate: 2021-10-04
+
+-   releaseCycle: "3.5"
+    releaseDate: 2015-01-13
+    eol: 2018-08-07
+    latest: '3.5.27'
+    latestReleaseDate: 2018-08-07
```

Comment:
According to http://www.squid-cache.org/Versions/v3/3.5/ the latest is:
> squid-3.5.28 | 15 Jul 2018

```suggestion
    latest: '3.5.28'
    latestReleaseDate: 2018-07-15
```

Reply:
6 is still in beta, there's no stable release yet. 6.0.3 is the latest beta.

Reply:
So the confusion comes because the `3.5.27` tag was created _before_ the `3.5.28` tag. The dates as per our automation are:

  "3.5.27": "2018-08-07",
  "3.5.28": "2018-07-16",


The `3.5.27` tag is definitely pointing to the wrong commit. It includes the [changes made for 3.5.28](https://github.com/squid-cache/squid/blob/SQUID_3_5_27/ChangeLog#L412-L427) as well.

Reply:
The correct commit seems to be https://github.com/squid-cache/squid/commit/b1268cb4604b5baab9a33db4202d6b3c464ac56d.